operations/jsonnet/microservices/config.libsonnet

```diff
@@ -35,6 +35,9 @@
     },
     vulture: {
       replicas: 0,
+      tempoPushUrl: 'http://distributor:14250',
+      tempoQueryUrl: 'http://query-frontend:3100',
+      tempoOrgId: '',
```

Comment:
Double-checking. We had org-id=1 and now it's empty. That's ok?

Reply:
yup :+1: